conans/test/unittests/model/build_info_test.py

```diff
@@ -209,3 +209,57 @@ def cpp_info_build_modules_test(self):
                              deps_cpp_info["myname"].build_modules_paths)
         self.assertListEqual([os.path.join(folder, "mod-release.cmake")],
                              deps_cpp_info["myname"].debug.build_modules_paths)
+
+    def deps_cpp_info_sysroot_test(self):
```

Comment:
Should these 2 tests be added before this PR? It seems they are testing some previous behavior and not changes of this PR?